bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/ActionImpl.java

```diff
@@ -0,0 +1,88 @@
+/*******************************************************************************
+ * Copyright (c) 1997, 2015 by ProSyst Software GmbH
+ * http://www.prosyst.com
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    ProSyst Software GmbH - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.smarthome.automation.core;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.eclipse.smarthome.automation.Action;
+import org.eclipse.smarthome.automation.Connection;
+import org.eclipse.smarthome.automation.Rule;
+import org.eclipse.smarthome.automation.handler.ActionHandler;
+import org.eclipse.smarthome.automation.type.ActionType;
+import org.eclipse.smarthome.automation.type.Input;
+import org.eclipse.smarthome.automation.type.Output;
+import org.eclipse.smarthome.config.core.ConfigDescriptionParameter;
+
+/**
+ * Actions are the part of "THEN" section of the {@link Rule} definition.
+ * Elements of this section are expected result of {@link Rule} execution. The
+ * Action can have {@link Output} elements. These actions are used to process
+ * input data as source data of other Actions. Building elements of actions ( {@link ConfigDescriptionParameter}s,
+ * {@link Input}s and {@link Output}s) are
+ * defined by {@link ActionType}
+ * 
+ * @author Yordan Mihaylov, dimova, v.ilchev
```

Comment:
Please add an author tag for every person individually and give their full name.